tap_postgres/sync_strategies/logical_replication.py

```diff
@@ -318,43 +327,45 @@ def locate_replication_slot(conn_info):
 
 
 def sync_tables(conn_info, logical_streams, state, end_lsn):
-    start_lsn = min([get_bookmark(state, s['tap_stream_id'], 'lsn') for s in logical_streams])
+    comitted_lsn = min([get_bookmark(state, s['tap_stream_id'], 'lsn') for s in logical_streams])
+    start_lsn = comitted_lsn
     time_extracted = utils.now()
     slot = locate_replication_slot(conn_info)
     last_lsn_processed = None
-    #When no data is received, poll every 5 seconds for 15 seconds total
-    poll_interval = 5.0
-    poll_total_seconds = conn_info['logical_poll_total_seconds'] or 15
-    begin_timestamp = datetime.datetime.now()
-    # When data is received, send feedback to keep connection open
-    feedback_interval = 5.0
-    feedback_timestamp = datetime.datetime.now()
+    logical_poll_total_seconds = conn_info['logical_poll_total_seconds'] or 1800
```

Comment:
Can we change it to `conn_info.get('logical_poll_total_seconds', 1800)` to get the default values without error?

`logical_poll_total_seconds` is optional in config and if it's not defined in `conn_info` then it throws exception:
```
KeyError: 'logical_poll_total_seconds'
```

Reply:
Please ignore this change requests. The key already has a value at this point of the code.

Default value set to 0 at the very beginning:
https://github.com/transferwise/pipelinewise-tap-postgres/blob/48eff3e8cc0235f3ee5f816bd256b42ad0287c45/tap_postgres/__init__.py#L691

Reply:
Fixed